[SDS-NNN] Minor improvements (#2)
QuTech-Delft/quantum-inspire-examples: README.md

jupyter notebook --notebook-dir="docs/notebooks"
```

> and select an IPython notebook (file with extension `ipynb`) to run from one of the directories.

Reviewer: ```suggestion
and select a Jupyter notebook (file with extension `ipynb`) to run from one of the directories.
```
ipython has been replaces/renamed by Jupyter. This holds on more locations, so perhaps just rename it one by one

Author: Also changed all the instances
